src/test/versionbits_tests.cpp

```diff
@@ -394,6 +444,16 @@ BOOST_AUTO_TEST_CASE(versionbits_computeblockversion)
             check_computeblockversion(chainParams->GetConsensus(), static_cast<Consensus::DeploymentPos>(i));
         }
     }
+
+    {
+        // Use regtest/testdummy to ensure we always exercise the
+        // min_lock_in_time test, even if we're not using that in a
+        // live deployment
+        ArgsManager args;
+        args.ForceSetArg("-vbparams", "testdummy:1199145601:1230767999:403200"); // January 1, 2008 - December 31, 2008, min act height 403200
```

Comment:
In e94fc819d8a996ce3690274c307c537329dfb95b "tests: test versionbits delayed activation"

How come `min_activation_height` is set so high? By my calculation, it only needs to be at least 576.

Reply:
To make sure that it doesn't blow up when a mainnet deployment is configured with a similarly high min_activation_height and it's tested.

Reply:
Should we not use 700000+? I can't see what would change between the two values, but if the goal is realism (maybe it runs too slowly :) )

Reply:
576 to 400k is a factor of 700x, 400k to 700k is only an extra 73% -- if the test blows up from 2s to 1400s that's a huge problem, if it goes from 2s to 3.5s, meh. :) So leaving it as is; though you're right, 700k would have been a fine choice.